TASVideos/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -56,6 +56,7 @@ public static IApplicationBuilder UseMvcWithOptions(this IApplicationBuilder app
 		]);
 		app.Use(async (context, next) =>
 		{
+			context.Response.Headers["Cross-Origin-Embedder-Policy"] = "unsafe-none"; // this is as unsecure as before, but can't use `credentialless`, see https://github.com/TASVideos/tasvideos/issues/1852

```

Comment:
If it is as unsecure as before, what is the value of adding this?

Reply:
Being explicit, and resolving warnings on securityheaders.com and the like.